lib/src/main/java/com/masoudss/lib/WaveformSeekBar.kt

```diff
@@ -248,6 +249,8 @@ open class WaveformSeekBar @JvmOverloads constructor(
     override fun onDraw(canvas: Canvas) {
         super.onDraw(canvas)
         sample?.let { waveSample ->
+            onCompletionListener?.onComplete()
```

Comment:
Please move this call to end of `onDraw` method.

Reply:
Sure